content/docs/start/experiments.md

```diff
@@ -18,17 +18,90 @@ once they're no longer needed.
 
 ## Running experiments
 
-Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
-[compare the changes](/doc/start/metrics-parameters-plots). Let's further
-increase the number of features in the `featurize` stage to see how it compares.
+In the previous page, we learned how to tune
+[ML pipelines](/doc/start/data-pipelines) and compare the changes. In this
+section we'll use a
+[Get Started to Experiments project](https://github.com/iterative/get-started-experiments)
+to illustrate experimentation features in DVC 2.0
+
+<details>
+
+### 🆕 Click here for the instructions to install the project
+
+These commands are run in
+[`get-started-experiments`](https://github.com/iterative/get-started-experiments)
+project. You can run these commands after cloning the repository and install the
+requirements.
+
+Please clone and create a virtual environment:
+
+```dvc
+$ git clone https://github.com/iterative/get-started-experiments
+$ cd get-started-experiments
+$ python -m venv .venv
+$ . .venv/bin/activate
+$ python -m pip install -r requirements.txt
+```
+
+Then you can `dvc pull` to get the dataset and run the commands in this
+document. For detailed information on parameters and the project structure
+please refer to the
+[project repository](https://github.com/iterative/get-started-experiments)
+
+</details>
 
 `dvc exp run` makes it easy to change <abbr>hyperparameters</abbr> and run a new
-experiment:
+experiment. We'll use it to search for parameters to increase the classification
+performance of our model using
+[Fashion-MNIST](https://github.com/iterative/get-started-experiments) dataset.
+
+In order to run a baseline experiment with the default parameters defined in
+`params.yaml`:
+
+```dvc
+$ dvc exp run
+'data/fashion-mnist/raw.dvc' didn't change, skipping
+Stage 'prepare' didn't change, skipping
+Stage 'preprocess' didn't change, skipping
+Stage 'train' didn't change, skipping
+Stage 'evaluate' didn't change, skipping
+```
+
+This resembles `dvc repro` without any command-line arguments. However, when
+using `dvc repro` we need to update `params.yaml` manually, run the pipeline, if
+the results are worth it commit them to DVC and Git. `dvc exp` automates this
+process through its subcommands.
+
+Let's see some metrics produced by this baseline experiment:
 
 ```dvc
-$ dvc exp run --set-param featurize.max_features=3000
+$ dvc exp show --include-metrics categorical_accuracy \
+               --include-params  model.name,model.cnn.conv_units
```

Comment:
Should we have a version of the project that has only these parameters and metrics instead of having to pass all these options to `dvc exp show`? It's a lot of parameters for the first time we introduce this command.

Reply:
A couple other suggestions to make the commands and tables easier to read:
* Would `accuracy` or `acc` be sufficient?
* Is`model.name` needed here if we are only using CNN?
* Do we need the nested parameter names (can it just be `conv_units` instead of `model.cnn.conv_units`)?

Reply:
I've updated the repo to contain only the following params and metrics will be limited to `(val_)acc` and `(val_)loss`.

```yaml
epochs: 1
conv_units: 16
dense_units: 128
dropout: 0.5
```

I'll renew all the commands and screenshots. @dberenbaum @jorgeorpinel